src/hookable.ts

```diff
@@ -113,10 +114,14 @@ export class Hookable <
   }
 
   callHook<NameT extends HookNameT> (name: NameT, ...args: Parameters<InferCallback<HooksT, NameT>>) {
+    return this.callHookWith((hooks, ...args) => serial(hooks, fn => fn(...args as any)), name, ...args)
```

Comment:
Let's keep `callHook` as is to avoid unnecessary wrapper function creation and calling